src/components/Calendar.scss

```diff
@@ -0,0 +1,162 @@
+.Calendar-heading, .Calendar-secondary-heading {
+  height: 60px;
+}
+.Calendar-title, .Calendar-month {
+  margin-right: $su-medium;
+  display: inline-block;
+}
+.Calendar-secondary-heading {
+  padding: $su-medium;
+  border-left: 1px solid $ts-grey;
+  border-right: 1px solid $ts-grey;
+  border-top: 1px solid $ts-grey;
+}
+.Calendar-row[data-ui-component="week-heading"] li {
+  display: none;
+}
+.Calendar-row[data-ui-component="week-heading"] {
+  background: $cu-background--light;
+  margin: 0;
+  padding: $su-small 0;
+  border: 1px solid $ts-grey;
+  color: $cu-primary;
+  list-style-type: none;
+}
+.Calendar-row[data-ui-component="week"] {
+  border-right: 1px solid $ts-grey;
+  & > [class*="Calendar-cell"] {
+    height: auto;
+    padding-bottom: $su-small;
+    border-left: 1px solid $ts-grey;
+    border-bottom: 1px solid $ts-grey;
+    span {
+      display: block;
+    }
+  }
+  & > [data-ui-component="current-day"] {
+    background: $cu-primary--bg;
+    span, abbr {
+      color: $cu-primary;
+      font-weight: $tu-bold-fontWeight;
+    }
+  }
+  & > [data-ui-component="next-month"] {
+    background: $cu-background--light;
+    span, abbr {
+      opacity: 0.6;
+    }
+  }
+}
+
+// responsive/widget
+
+.Calendar-widget--exampleWrapper {
+  width: 300px;
+}
+
+.Calendar.Calendar-widget {
+  .Calendar-secondary-heading {
+    text-align: center;
+    .Button--paginate {
+      float: left;
+      &:last-child {
+        float: right;
+      }
+    }
+
+  }
+  .Calendar-row[data-ui-component="week-heading"] li {
+    display: block;
+  }
+  .Calendar-row[data-ui-component="week"] {
+    & > [class*="Calendar-cell"] {
```

Comment:
Remove nesting if possible.

Reply:
<img width="322" alt="screen shot 2017-10-10 at 12 30 39 pm" src="https://user-images.githubusercontent.com/9888467/31401070-47e35e6e-adb7-11e7-9b6a-5060d8c31d8c.png">



The weekday names should be cropped to the first two letters in the widget view. This is probably better achieved in the markup, though. Something like:
```
<li class="Calendar-cell">
  Mo<span class="u-hiddenVisually">nday</span>
</li>
```
This way the full weekday will still be legible by screenreaders but cropped visually.

Reply:
Yes! I was going to do just that, but I had hesitations about that for legibility purposes. Is that legal markup in that it won't mess up screen readers and whatnot? This was actually a place where I wanted to discuss a little bit.

Reply:
With this, what do you think about using the <abbr> element and replacing with the abbreviated text with the full text at larger screens?

Never a fan of breaking up words with spans.. something along these lines maybe : https://blog.escapecreative.com/how-to-replace-text-with-css/

Then for the content you just pull from the title

Reply:
Seems a little complicated to me... what's the issue with breaking up a word?

Reply:
Hmm.. maybe to move the complication to the CSS rather than the markup?